--- /dev/null
+++ b/build/config/siso/backend_config/nativelink.star
@@ -0,0 +1,20 @@
+# -*- bazel-starlark -*-
+# Copyright 2026 The Helium Authors
+# You can use, redistribute, and/or modify this source code under
+# the terms of the GPL-3.0 license that can be found in the LICENSE file.
+
+load("@builtin//struct.star", "module")
+
+def __platform_properties(ctx):
+    properties = {
+        "cpu_count": "1",
+    }
+    return {
+        "default": properties,
+        "large": properties,
+    }
+
+backend = module(
+    "backend",
+    platform_properties = __platform_properties,
+)
