--- a/chrome/browser/ui/views/profiles/profile_menu_view.cc
+++ b/chrome/browser/ui/views/profiles/profile_menu_view.cc
@@ -998,6 +998,7 @@ void ProfileMenuView::MaybeBuildBatchUpl
 }
 
 void ProfileMenuView::BuildAutofillSettingsButton() {
+  return;
   CHECK(!profile().IsGuestSession());
 
   int message_id = IDS_PROFILE_MENU_AUTOFILL_SETTINGS_BUTTON;
@@ -1097,6 +1098,7 @@ void ProfileMenuView::MaybeBuildGoogleSe
 }
 
 void ProfileMenuView::MaybeBuildManageGoogleAccountButton() {
+  return;
   CHECK(!profile().IsGuestSession());
   signin::IdentityManager* identity_manager =
       IdentityManagerFactory::GetForProfile(&profile());
